### DISCUSSION TITLE
How to use IME typing Chinese in Menu-Search Bar?

### DISCUSSION THREAD

**Discussion by baozidai** [linuxmint/discussions#26] [Questions]
# Does anyone know how to type Chinese in this search bar?
My IME is Fcitx. Thanks. :pray: 
![Canttype](https://github.com/linuxmint/discussions/assets/8564451/40b86e3f-3a6a-4af1-bf08-f59bab6779ce)

# More
If cannot Chinese, I guess also neither Korean or Japanese.

**Comment by fredcw** [comment]
Ibus instead of fcitx seems to work with japanese (kana) language but I can't find chinese on the list of languages. Korean is there but it doesn't seem to do anything.

btw, there's also another open issue for this https://github.com/linuxmint/cinnamon/issues/11713
